File: pset/pseudocode/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //gets value from user
  
    int n;

    do
    {
        n =  get_int("input height: ");
    }


    while (n < 1 || n > 8);

// prints  hash triangle to the right

     
    for (int i = 0; i < n; i++)
    {
        for (int k = n - i ; k > 0 ; k--)
        {
            printf(" ") ; 
        }
            
        for (int j = 0 ; j <= i ; j++)
        {
            printf("#");
        }

        printf("\n");
    }


}

File: pset6/mario.py
import cs50 
# gets user input and prints pyramid 

height = cs50.get_int("Height: ")
while height < 1 or height > 8:
    height = cs50.get_int("Height: ")
for i in range(height):
    #print("-", end="")
    for k in range(height - i - 1):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    for k in range(1):
        print("  ", end="")
        for k in range(i + 1):
            print("#", end="")
    print()